DCOS/templates/preprovision/extensions/preprovision-agent-windows/v1/preprovision-agent-windows-fluentd.ps1

```diff
@@ -0,0 +1,65 @@
+$here = Split-Path -Parent $MyInvocation.MyCommand.Path
+$TD_AGENT = "td-agent-3.1.1-0-x64.msi"
+$url = "http://packages.treasuredata.com.s3.amazonaws.com/3/windows/$TD_AGENT"
+$MSI_PATH = Join-Path $here $TD_AGENT
+$fluentPath = "$env:SystemDrive\opt\td-agent\embedded\bin"
+$env:Path = "$fluentPath;" + $env:Path.Replace("$fluentPath;", "")
+
+function Get-Fluentd () {
+    Write-Host "Downloading fluentd/td-agent..."
```

Comment:
Since you moved the script, please change all the occurrences of `Write-Host` to `Write-Output`. See my comment above. The only place where you don't need to do this is in the Pester `*.Tests.ps1` PowerShell scripts.